server/src/domain/Skill.ts

```diff
@@ -0,0 +1,17 @@
+import { Column, Entity } from "typeorm";
+
+import { BaseModel } from "../infrastructure/BaseModel";
+
+export enum SkillType {
+  Tecnology,
+  DesignTools,
```

Comment:
Can we have these singular?

```suggestion
  DesignTool,
```

Reply:
Sure!